src/ucp/core/ucp_worker.c

```diff
@@ -2227,7 +2228,7 @@ ucs_status_t ucp_worker_create(ucp_context_h context,
     return status;
 }
 
-static void ucp_worker_put_flush_req(ucp_request_t *req)
+static void ucp_worker_discard_uct_ep_complete(ucp_request_t *req)
```

Comment:
why need to rename

Reply:
`ucp_worker_put_flush_req()` was used for flush, but now it is not the case

Reply:
then maybe simplify like `ucp_request_discard_complete`? and move to ucp_request.c

Reply:
better to not move this code to the common code